app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
+        try:
+            cafe.visit_cafe(friends[i])
+        except (NotVaccinatedError, OutdatedVaccineError):
+            return 'All friends should be vaccinated'
```

Comment:
it's better to except separate error end return error messages. This code will be more maintainable.

```suggestion
        except NotVaccinatedError as error:
            return error
        except OutdatedVaccineError as error:
            return error
```